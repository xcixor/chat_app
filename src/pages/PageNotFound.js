import React from "react";
import { Container } from "react-bootstrap";

export default function PageNotFound() {
	return (
		<Container>
			<h2>Sorry, we could not find the page you are looking for.</h2>
		</Container>
	);
}
