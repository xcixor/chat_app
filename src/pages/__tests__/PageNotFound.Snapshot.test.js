import React from "react";
import PageNotFound from "../PageNotFound";
import renderer from "react-test-renderer";

it("Renders correctly", () => {
	const tree = renderer.create(<PageNotFound />);
	expect(tree).toMatchSnapshot();
});
