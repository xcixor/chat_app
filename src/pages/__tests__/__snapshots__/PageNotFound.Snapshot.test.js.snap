// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Renders correctly 1`] = `
<div
  className="container"
>
  <h2>
    Sorry, we could not find the page you are looking for.
  </h2>
</div>
`;
